button-examples/Web/web-button-example.md

```diff
@@ -96,11 +99,11 @@ Add an icon to your `mdc-button` instance using the following steps:
     </button>
     ```
 
-    **Example using SVG Icons**
+    **Example using SVG Icon**
     ```html
     <button class="mdc-button">
       <div class="mdc-button__ripple"></div>
-      <svg class="mdc-button__icon" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="...">
+      <svg class="mdc-button__icon" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="..."> // I feel like this example should be more complete -- make an actual shape so users can copy and paste it and see something
```

Comment:
I was nervous because the code block for MD would use a really long scroll bar: I forget if HTML handles carriage returns within a tag.

We can add it back in.

Reply:
...Okay, it was never a complete example, and I copied/pasted from the original web article:

https://material.io/develop/web/components/buttons/#icons

Can you provide sample code and a screenshot?